compiler/test/Dialect/Mesh/ops.mlir

```diff
@@ -0,0 +1,24 @@
+// RUN: byteir-opt %s | FileCheck %s
+
+mesh.cluster @mesh0(rank = 3, dim_sizes = [4, 8, 12])
```

Comment:
is mesh.cluster per module?

Reply:
mesh.cluster is defined inside a module op and there could be multiple mesh.cluster op per module.
It should be useful for pipeline parallel, maybe I should supplement some design doc for it.

Reply:
or one example for pp or multiple mesh.cluster's will be good.